stage1/init/init.go

```diff
@@ -341,8 +341,9 @@ func getArgsEnv(p *Pod, flavor string, systemdStage1Version string, debug bool)
 	args = append(args, "--")
 	args = append(args, "--default-standard-output=tty") // redirect all service logs straight to tty
 	if !debug {
-		args = append(args, "--log-target=null") // silence systemd output inside pod
-		args = append(args, "--show-status=0")   // silence systemd initialization status output
+		args = append(args, "--log-target=null")   // silence systemd output inside pod
+		args = append(args, "--log-level=warning") // limit log output (systemd-shutdown ignores --log-target)
```

Comment:
Don't know the reasoning, but it's here http://cgit.freedesktop.org/systemd/systemd/tree/src/core/main.c?id=v220#n2003